Documentation/platforms/container-linux-systemd.md

```diff
@@ -0,0 +1,164 @@
+# Run etcd on Container Linux with systemd
+
+The following guide shows how to run etcd with [systemd][systemd-docs] under [Container Linux][container-linux-docs].
+
+## Provisioning an etcd cluster
+
+Cluster bootstrapping in Container Linux is simplest with [Ignition][container-linux-ignition]; `coreos-metadata.service` dynamically fetches the machine's IP for discovery. Note that etcd's discovery service protocol is only meant for bootstrapping, and cannot be used with runtime reconfiguration or cluster monitoring.
+
+To avoid accidental misconfiguration, [Container Linux Config Transpiler][container-linux-ct] helpfully verifies etcd configurations when generating Ignition files:
+
+```bash
+# passing wrong etcd configuration to 'ct'
+cat > /tmp/ct-etcd.cnf <<EOF
+etcd:
+  version: 3.1.8
+  name: s1
+  data_dir_x: /var/lib/etcd
+  advertise_client_urls:       http://{PUBLIC_IPV4}:2379
+  initial_advertise_peer_urls: http://{PRIVATE_IPV4}:2380
+  listen_client_urls:          http://0.0.0.0:2379
+  listen_peer_urls:            http://{PRIVATE_IPV4}:2380
+  discovery:                   https://discovery.etcd.io/<token>
+EOF
+ct --platform=gce --in-file /tmp/ct-etcd.cnf > /tmp/config.ign
+
+# 'ct' verifies etcd configurations
+warning at line 3, column 2
+Config has unrecognized key: data_dir_x
+```
+
+```bash
```

Comment:
working example should probably be put before the broken example and should have a description:
The [Container Linux Config Transpiler][container-linux-ct] compiles etcd configuration files into Ignition configuration files: